### PR TITLE
Project 2: Add and remove items

### PR DESCRIPTION
**Pull request by binhussen**
## Project 2: Add and remove items.
In this milestone, we have done
- Implement a function for adding a new task (add a new element to the array).
- Implement a function for deleting a task (remove an element from the array).
- Implement a function for editing task descriptions.
- Deleting a task should update all remaining items' indexes,
- Implement a function for a checkbox to complete tasks.
- All changes to the To Do List should be saved in local storage.

### CODE REVIEW

File: src/operation.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    e.preventDefault();
+    const task = {
+      index: this.tasks.length,
+      description: input.value,
+      completed: false,
+    };
+    this.tasks.push(task);
+    setLocalStorage(this.tasks);
+    this.displayTasks();
+  };
```
Comment (Bondok6) [inline src/operation.js:24]: - [OPTIONAL] I suggest clearing the input after you add a new task by adding this line `input.value = ''; ` 💯

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @binhussen 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- No Linter errors ✅


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions
- [OPTIONAL] I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by binhussen** [comment]
Thank you @Bondok6 for your feedback and approval.

File: src/operation.js (hunk 1 above)
Comment (binhussen) [reply src/operation.js:24]: Thank you @Bondok6 for your feedback, I will Modify It.

**Pull request merged**